test/plugin/test_elasticsearch_error_handler.rb

```diff
@@ -11,15 +11,15 @@ class TestPlugin
     def initialize(log)
       @log = log
       @write_operation = 'index'
-      @error_events = Fluent::MultiEventStream.new
+      @error_events = []
```

Comment:
why change from eventstream?

Reply:
Needed to confirm we are receiving something that responds to `backtrace`.  Given nothing in the test relies specifically on a MultiEventStream I saw no harm in modifying it.